File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        lst =s.split()
        s1 = ""
        for i in range(len(lst)-1):
            s1 += lst[-i-1]
            s1+= " "
        s1+= lst[0]
        return s1

File: 0151-reverse-words-in-a-string/0151-reverse-words-in-a-string.cs
using System.Text.RegularExpressions;
public class Solution {
    public string ReverseWords(string s) {
        string[] words = s.Split(' ');
        Array.Reverse(words);
        string result = string.Join(' ', words);
        string cleanedString = Regex.Replace(result.Trim(), @"\s+", " ");
        return cleanedString;
    }
}